apps/indexer/src/api/services/proposals/proposals.ts

```diff
@@ -104,4 +119,47 @@ export class ProposalsService {
 
     return { ...proposal, status };
   }
+
+  /**
+   * Returns the delegates with active delegations that didn't vote on a given proposal
+   */
+  async getProposalNonVoters(
+    proposalId: string,
+    skip: number = 0,
+    limit: number,
+    orderDirection: "asc" | "desc",
+    addresses?: Address[],
+  ): Promise<VotersResponse> {
+    const nonVoters = await this.proposalsRepo.getProposalNonVoters(
+      proposalId,
+      skip,
+      limit,
+      orderDirection,
+      addresses,
+    );
+
+    const _addresses = addresses ? addresses : nonVoters.map((v) => v.voter);
+
+    const comparisonTimestamp = Math.floor(Date.now() / 1000 - DaysEnum["30d"]);
```

Comment:
The following snippet:

```ts
Math.floor(Date.now() / 1000)
```

Is repeated quite a bit around the codebase; a couple of questions:

1. Why is this necessary? Does it have to do with the way `drizze` serializes/deserializes timestamps?
2. Wouldn't it be better to extract this logic to a helper function in an `utils` file somewhere?